Sources/Preferences/PreferencesTabViewController.swift

```diff
@@ -31,17 +34,15 @@ final class PreferencesTabViewController: NSViewController, PreferencesStyleCont
 	}
 
 	func changePreferencesStyle(to newStyle: PreferencesStyle) {
-		changePreferencesStyleController(preferences: self.preferencePanes, style: newStyle)
-	}
-
-	private func changePreferencesStyleController(preferences: [PreferencePane], style: PreferencesStyle) {
 		let toolbar = NSToolbar(identifier: "PreferencesToolbar")
 		toolbar.allowsUserCustomization = false
 		toolbar.displayMode = .iconAndLabel
 		toolbar.showsBaselineSeparator = true
 		toolbar.delegate = self
 
-		switch style {
+		let preferences = self.preferencePanes
```

Comment:
I would just use `self.preferencePanes` or rather `preferencePanes` directly in the `SegmentedControlStyleViewController`. Maybe also rename it's argument from `preferences` to `preferencePanes`.